docs/ops/movement/Concat_1.md

```diff
@@ -11,7 +11,7 @@
 * *axis*
 
   * **Description**: *axis* specifies dimension to concatenate along
-  * **Range of values**: integer number. Negative value means counting dimension from the end
+  * **Range of values**: integer number. Negative value means counting dimension from the end. Range is '[-R, R-1]' where 'R' is the rank of all inputs.
```

Comment:
Minor edits are required:

Range is '[-R, R-1]' where 'R' is the rank of all inputs. --> The range is '[-R, R-1]', where 'R' is the rank of all inputs.